internal/ingest/ingest.go

```diff
@@ -25,6 +26,9 @@ type Configs struct {
 	StartLedger          int
 	EndLedger            int
 	LogLevel             logrus.Level
+	AppTracker           sentry.SentryTracker
+	SentryDsn            string
+	StellarEnvironment   string
```

Comment:
I'd suggest we only have the app tracker here. Then on the config we only use a local variable:

```go
var sentryDSN string

utils.SentryOptions(&sentryDSN)
```

Reply:
Done